addons/sale/models/sale.py

```diff
@@ -150,7 +147,7 @@ def _compute_tax_id(self):
         ('invoiced', 'Fully Invoiced'),
         ('to invoice', 'To Invoice'),
         ('no', 'Nothing to Invoice')
-        ], string='Invoice Status', compute='_get_invoiced', store=True, readonly=True)
+    ], string='Invoice Status', compute='_get_invoiced', store=True, readonly=True)
```

Comment:
probably revert this

Reply:
Globally you're always right when you plan to revert something ;)